scripts/generate-weights.sh

```diff
@@ -2,22 +2,18 @@
 
 # Executes and replaces all benchmarks with the new weights
 
-echo "Benchmarking proposals_discussion..."
-./target/release/joystream-node benchmark --pallet=proposals_discussion --extrinsic=* --chain=dev --steps=50 --repeat=20 --execution=wasm --output=. > /dev/null
-mv proposals_discussion.rs runtime/src/weights/
-echo "proposals_discussion benchmarked"
+echo "Generating weights..."
 
-echo "Benchmarking proposals_engine..."
-./target/release/joystream-node benchmark --pallet=proposals_engine --extrinsic=* --chain=dev --steps=50 --repeat=20 --execution=wasm --output=. > /dev/null
-mv proposals_engine.rs runtime/src/weights/
-echo "proposals_engine benchmarked"
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+$SCRIPT_DIR/../target/release/joystream-node benchmark \
+    --pallet=* \
```

Comment:
This command gets stuck with pallet=*. I waited for 40-50min. But it worked with single pallet in current build (I tried pallet_constitution).

Reply:
I think this is the fault of `pallet_im_online` with just 5 steps and 2 repeats it takes 44seconds in my computer. If you reduce in this script to 5 steps and 2 repeats it'll probably finish in around a minute.

And if we extrapolate this to 50 steps with 20 repeats(around 100 times more) it would take 440 seconds, meaning ~1 and a half hours. I'll leave it running today to confirm how much it'd take to finish.